### PR TITLE
Update db structure

### PR DESCRIPTION
**Pull request by JohannDallmann**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/backend/service/user/GenerateUserCityCollectionService.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    public String generateUserCityCollection(ImportMongoUserDTO newUserWithoutId){
+        String newUUID = generateUUIDService.generateUUID();
+        String newUUIDforUnknownCity = generateUUIDService.generateUUID();
+        UserCity defaultUserCity = citiesRepo.findUserCityByCityName(newUserWithoutId.getHomecity())
+                .orElse(new UserCity(newUUIDforUnknownCity, "unknown City", 52.4, 13.29));
+        Map<String, UserCity> newUserCityMap = new HashMap<>();
+        newUserCityMap.put(defaultUserCity.getCityId(), defaultUserCity);
+        UserCityCollection newUserCityCollection =new UserCityCollection(newUUID,newUserCityMap);
+        cityCollectionRepo.save(newUserCityCollection);
```
Comment (MJaschewski) [inline backend/src/main/java/de/neuefische/backend/service/user/GenerateUserCityCollectionService.java]: save already returns the new object so you can just return the save method and skip the next line.

File: backend/src/main/java/de/neuefische/backend/service/user/GenerateUserFriendCollectionService.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+public class GenerateUserFriendCollectionService {
+    private final FriendCollectionRepo friendCollectionRepo;
+    private final GenerateUUIDService generateUUIDService;
+
+    public String generateUserFriendCollection(){
+        String newUUID = generateUUIDService.generateUUID();
+        Map<String, Friend> newUserFriendMap = new HashMap<>();
+        UserFriendCollection newUserFriendCollection = new UserFriendCollection(newUUID,newUserFriendMap);
+        friendCollectionRepo.save(newUserFriendCollection);
+        return newUserFriendCollection.getId();
```
Comment (MJaschewski) [inline backend/src/main/java/de/neuefische/backend/service/user/GenerateUserFriendCollectionService.java]: see above

File: backend/src/main/java/de/neuefische/backend/service/user/GenerateUserLocationCollectionService.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+@RequiredArgsConstructor
+public class GenerateUserLocationCollectionService {
+    private final LocationCollectionRepo locationCollectionRepo;
+    private final GenerateUUIDService generateUUIDService;
+
+    public String generateUserLocationCollection(){
+        String newUUID = generateUUIDService.generateUUID();
+        Map<String, Location> newUserLocationMap = new HashMap<>();
+        UserLocationCollection newUserLocationCollection = new UserLocationCollection(newUUID,newUserLocationMap);
+        locationCollectionRepo.save(newUserLocationCollection);
```
Comment (MJaschewski) [inline backend/src/main/java/de/neuefische/backend/service/user/GenerateUserLocationCollectionService.java]: see above

**Review by MJaschewski — CHANGES_REQUESTED** [review]
looks good! I would recommend some changes!

**Comment by JohannDallmann** [comment]
thank you for the advice. i changed it.

**Review by MJaschewski — APPROVED** [review]

**Pull request merged**